Project-I-Social-Card/social-card/src/components/FooterComponents/FooterIcons.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Footer.css';
+
+const FooterIcons = props => {
+    return (
+        <div className='iconContainer'>
```

Comment:
Footer component can be much DRYer:
```suggestion
    <div className="iconContainer">
      <i className="far fa-comment" />
      <i className="fas fa-retweet"><span>{props.retweets}</span></i>
      <i className="far fa-heart" ><span>{props.likes}</span></i> 
      <i className="far fa-envelope" />
    </div>
```